### DISCUSSION TITLE
Compatiblity with Journals plugin

### DISCUSSION THREAD

**Discussion by RobTemple** [ivan-lednev/obsidian-day-planner#604] [Q&A] [answered]
Are there settings to allow compatibility with the Journals plugin?

**Comment by ivan-lednev** [comment, marked as answer]
Hello, @RobTemple, this should be a feature request. Please create one and describe:
1. What the journals plugin is
2. What kind of compatibility you'd like to see
3. Why is it valuable to you and other users

[Here is a template link](https://github.com/ivan-lednev/obsidian-day-planner/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.yaml).
